src/main/java/com/aws/greengrass/easysetup/DeviceProvisioningHelper.java

```diff
@@ -142,6 +147,7 @@ public DeviceProvisioningHelper(String awsRegion, String environmentStage, Print
                 : EnvironmentStage.fromString(environmentStage);
         this.iotClient = IotSdkClientFactory.getIotClient(awsRegion, envStage);
         this.iamClient = IamSdkClientFactory.getIamClient(awsRegion);
+        this.stsClient = StsSdkClientFactory.getStsClient(awsRegion);
```

Comment:
Update the required permission for e2e tests in readme https://github.com/aws-greengrass/aws-greengrass-nucleus/blame/main/DEVELOPING.md#L51-L57